portfolio/src/main/webapp/index.html

```diff
@@ -101,9 +97,20 @@ <h1>Gallery</h1>
             </div>
             <!--Below is the code for the "Interests" page. Due to time constraints, this page isn't implemented but the code is left here so that I can complete
                 later.-->
-            <div id="interests">
-              <h1>Interests and Hobbies</h1>
-              <p>Here are a few of my hobbies and interests!</p>
+            <div id="comments">
+              <h2>Comments</h2>
+              <p>Feel free to leave comments below!</p>
+              <form action="/data" method="POST">
+                <p>Name</p>
+                <input type="text" name="user-name">
+                <br/>
+                <p>Comment</p>
+                <input type="text" name="user-comment">
+                <br/> <br/>
```

Comment:
I'd recommend using a css style (e.g. padding-bottom, margin-bottom) to create this space instead. Break lines (i.e. `<br>`) spacing is based on the line-height which can differ based on font, browser, etc. so they can end up being quite finicky. Break lines are better used to represent something semantic (e.g. breaking off the text in a paragraph)